PathPlanning/ClothoidalPaths/clothoidal_paths.py

```diff
@@ -0,0 +1,144 @@
+"""
+Clothoidal Path Planner
```

Comment:
Could you please add a unit test file and documentation file based on this doc? :[How to contribute — PythonRobotics documentation](https://pythonrobotics.readthedocs.io/en/latest/how_to_contribute.html#adding-a-new-algorithm-example)